packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -39,24 +42,48 @@ export default function Onboarding() {
   ];
 
   const title: string = t('title');
+  const titlebis: string = t('titlebis');
   const description: string = t('description');
+  const descBis: string = t('descriptionBis');
```

Comment:
You could inline all of this directly, no need to create variables in my opinion

Reply:
Do not resolve comment without answering them or applying what's commented please